operator/pkg/ciliumendpointslice/endpointslice.go

```diff
@@ -110,11 +113,16 @@ func (c *Controller) Start(ctx cell.HookContext) error {
 	c.logger.Info("Bootstrap ces controller")
 	c.context, c.contextCancel = context.WithCancel(context.Background())
 	defer utilruntime.HandleCrash()
-	if c.slicingMode == cesIdentityBasedSlicing {
+
+	switch c.slicingMode {
+	case identityMode, "cesSliceModeIdentity":
```

Comment:
nit: Can we add comment here that `cesSliceModeIdentity` and `cesSliceModeFCFS` are accepted for backwards-compatibility? 

Ideally we'd even warn the user that they are using old values and need to upgrade their configuration. That allows us to eventually remove those cases

Reply:
Good suggestion! Added this :+1: